src/overlays/actors/ovl_Bg_Hakugin_Elvpole/z_bg_hakugin_elvpole.c

```diff
@@ -30,17 +30,111 @@ const ActorInit Bg_Hakugin_Elvpole_InitVars = {
     (ActorFunc)BgHakuginElvpole_Draw,
 };
 
-#endif
+// #pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Hakugin_Elvpole/BgHakuginElvpole_Init.s")
+void BgHakuginElvpole_Init(Actor* thisx, PlayState* play) {
+    s32 switchFlags;
```

Comment:
I would not be hugely surprised if this temp can be removed.

Reply:
it can, but it looks a lot uglier and unnecessarily bloated, in my opinion, but I can change it if that would be better

Reply:
My suspicion is that it's taking up the space of the GameState recast on the stack. And quite a lot of the time when there's a params access macro involved they don't seem to have a temp. If you feel that strongly I won't force you to, but that's what it seems to me is more likely.

Reply:
nah, I'll change it, its fine

Reply:
It seems like I was wrong, or forgot to save before compiling, because it doesn't actually match without a pad. which makes me think the variable is real?

Reply:
Just a normal `s32 pad` would be fine, and kind of expected.